lib/vmdb/loggers/request_logger/logdev.rb

```diff
@@ -0,0 +1,17 @@
+module Vmdb::Loggers
+  class RequestLogger
+    class Logdev
+      attr_reader :resource_id
+
+      def initialize(resource_id)
+        @resource_id = resource_id
+      end
+
+      def write(message)
```

Comment:
Does the return value of write matter?

Reply:
No at least not currently, `Logger#add` ignores the return value of `@logdev.write` and return` true`

I could see returning `message.length` to simulate what `File#write` would return if we wanted to try to match that

Reply:
No I just want to make sure we match what logdev.write does but really only if it matters.